### DISCUSSION TITLE
Disabling features

### DISCUSSION THREAD

**Discussion by JxxIT** [liedekef/events-made-easy#488] [Q&A] [answered]
Hi! First off, kudos for the amazing tool!
Is it currently possible to disable features entirely? I want to disable features like RSS and locations.

Thanks!

**Comment by liedekef** [comment, marked as answer]
While some features can be disabled, the rss feed cannot currently. And locations is even more difficult since it is very tightly integrated with events. Adding something to disable rss shouldn't be that difficult to do.

**Reply by JxxIT** [reply]
Thanks!
